src/lib/Chart.tsx

```diff
@@ -25,7 +25,7 @@ import { useBoundsContext } from "./BoundsManager";
 import { Manipulator } from "./Manipulator";
 import { DeepPartial, noop } from "ts-essentials";
 import deepmerge from "deepmerge";
-import { divSize, Size } from "./basic-types";
+import {Bounds, divSize, Size} from "./basic-types";
```

Comment:
Could you please enable automatic Prettier for this project? :)

We certainly need to configure pre-commit...